### PR TITLE
update all outputs for dealing with tags and source

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Issif <issif+github@gadz.org>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area config

/area outputs

> /area tests

**What this PR does / why we need it**:

Since Falco 0.31, new fields `source` and `tags` have been added in json events. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -25,4 +22,4 @@ linters:
   include:
   - EXC0002 # include "missing comments" issues from golint
   max-issues-per-linter: 0
-  max-same-issues: 0
+  max-same-issues: 0
```
Comment (cpanato) [inline .golangci.yml:25]: nit: new line

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,16 @@
+ARG BUILDER_IMAGE=golang:1.18-buster
 ARG BASE_IMAGE=alpine:3.15
+
```
Comment (cpanato) [inline Dockerfile:3]: this will break the goreleaser

Comment (Issif) [reply Dockerfile:3]: why?

File: Makefile [hunk 3]
```diff
@@ -41,7 +41,7 @@ TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/bin)
 GO_INSTALL = ./hack/go_install.sh
 
 # Binaries.
-GOLANGCI_LINT_VER := v1.43.0
+GOLANGCI_LINT_VER := v1.44.2
```
[the remark this answers is not in the document]
Comment (Issif) [reply Makefile:44]: I'll update it, thanks

File: .golangci.yml (hunk 1 above)
Comment (Issif) [reply .golangci.yml:25]: I'll add it :+1:

File: Dockerfile (hunk 2 above)
Comment (cpanato) [reply Dockerfile:3]: becasue goreleaser uses the binary that is built to inject that in the image, and this update will try to run the full dockerfile but goreleaser does not expected that https://goreleaser.com/customization/docker/

Comment (Issif) [reply Dockerfile]: got it

File: .circleci/config.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   default:
     docker:
-    - image: cimg/go:1.17.9
+    - image: cimg/go:1.18.1
```
Comment (cpanato) [inline .circleci/config.yml:5]: go 1.18.2 is out, it is just a matter of time to have this image updated. fyi

Comment (Issif) [reply .circleci/config.yml:5]: for a next PR :wink:

**Review by cpanato — APPROVED** [review]
cool
/lgtm

**Pull request merged**